cityscrape-config.sh

```diff
@@ -0,0 +1,16 @@
+# Variables for the cityscrape
```

Comment:
Something like 'Global settings' might be good here, as it implies these could be imported in the scope of any python module that'd run in the project after you source the file in the run script